packages/tocco-ui/src/EditableValue/typeEditors/IntegerEdit.js

```diff
@@ -0,0 +1,41 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import {intlShape} from 'react-intl'
+
+import NumberEdit from './NumberEdit'
+
+const IntegerEdit = props => {
+  const handleChange = value => {
+    if (props.onChange) {
+      props.onChange(value)
+    }
+  }
+
+  return (
+    <NumberEdit
```

Comment:
Hm so the IntegerEdit is exactly the same as Number Edit? Does it make sense to set PostPoint Digits according to the prop options.postPointDigits. With integer is always 0.

Reply:
The props already set this behaviour. If minValue and maxValue are set, as they are in the model for integers, the decimalScale is set to zero.